src/Components/Users/users.jsx

```diff
@@ -1,44 +1,101 @@
 import React from 'react';
+import * as axios from 'axios';
 
-import { I18N } from '../../constants';
-import usersArray from '../../usersArray';
+import { I18N, API } from '../../constants';
 
 import style from './users.module.css';
 
-const users = (props) => {
-  if (props.users.length === 0) {
-    props.setUsers(usersArray);
+import defaultImg from '../Image/person.jpg';
+
+class Users extends React.Component {
+  state = {
+    pages: [],
+  };
+  componentDidMount() {
+    const {
+      setUsers, setTotalUsersCount, currentPage, pageSize,
+    } = this.props;
+
+    axios.get(`${API.BASE_URL}${API.USER_RESOURCE}${API.PAGE}${currentPage}${API.COUNT}${pageSize}`)
+      .then(({ data }) => {
+        const { items, totalCount } = data;
+        setUsers(items);
+        setTotalUsersCount(totalCount);
+        this.a();
+      });
+  }
+  a = () => {
```

Comment:
Consider using more meaningful names for variables.

Reply:
Sure.I hurried